daemon/containerd/service.go

```diff
@@ -120,10 +119,18 @@ func (i *ImageService) StorageDriver() string {
 	return i.snapshotter
 }
 
-// ReleaseLayer releases a layer allowing it to be removed
+// ReleaseLayer releases the snapshot allowing it to be removed.
 // called from delete.go Daemon.cleanupContainer(), and Daemon.containerExport()
-func (i *ImageService) ReleaseLayer(rwlayer layer.RWLayer) error {
-	return errdefs.NotImplemented(errors.New("not implemented"))
+func (i *ImageService) ReleaseLayer(ctx context.Context, ctr *container.Container) error {
+	ls := i.client.LeasesService()
+	lease := leases.Lease{
+		ID: ctr.ID,
+	}
+	err := ls.Delete(ctx, lease, leases.SynchronousDelete)
```

Comment:
nit; I guess we can inline all of the above here (I _think_ that's still readable, even if we inline the client?)

```suggestion
	err := i.client.LeasesService().Delete(ctx, leases.Lease{ID: ctr.ID}, leases.SynchronousDelete)
```